rclpy/rclpy/impl/_rclpy_pybind11.pyi

```diff
@@ -333,3 +333,30 @@ class WaitSet(Destroyable):
 
     def wait(self, timeout: int) -> None:
         """Wait until timeout is reached or event happened."""
+
+
+
+# signal_handler
+class SignalHandlerOptions(Enum):
+    _value_: int
+    NO = ...
+    SigInt = ...
+    SigTerm = ...
+    All = ...
+
+
+def register_sigint_guard_condition(guard_condition: GuardCondition) -> None:
+    """Register a guard condition to be called on SIGINT."""
+
+def unregister_sigint_guard_condition(guard_condition: GuardCondition) -> None:
+    """Stop triggering a guard condition when SIGINT occurs."""
+
+def install_signal_handlers(options: SignalHandlerOptions) -> None:
+    """Install rclpy signal handlers."""
+
+def get_current_signal_handlers_options() -> SignalHandlerOptions:
+    """Get currently installed signal handler options."""
+
+def uninstall_signal_handlers() -> None:
+    """Uninstall rclpy signal handlers."""
+# signal_handler
```

Comment:
Nit, what is this comment? can we remove this?

```suggestion
# signal_handler
```

Reply:
Sure. I was using those comments as a way to group which cpp file the fuctions and class came from since the file is getting more complicated but, I will remove them.